docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -45,11 +45,153 @@ The remote service must implement the [Expo Updates protocol](/technical-specs/e
+
+For example, in a project that has the following in its app config:
+
+```json
+{
+  "expo": {
+    "runtimeVersion": {
+      "policy": "appVersion"
+    },
+    "version": "1.0.0",
+    "ios": {
+      "buildNumber": "1"
+    },
+    "android": {
+      "versionCode": 1
+    }
+  }
+}
+```
+
+The `"appVersion"` policy will set the runtime version to the project's current `"version"` property. The runtime version for the Android and iOS builds and any updates would be in this case `"1.0.0"`.
+
+This policy is great for projects that contain custom native code and that update the `"version"` field after every public release. To submit an app, the app stores require an updated native version number for each submitted build, which makes this policy convenient if you want to be sure that every version installed on user devices has a different runtime version.
+
+When using this policy, you need to manually update `"version"` field in app config every time there is a public release, but for Play Store's Internal Test Track and the App Store's TestFlight uploads, you can rely on `"autoIncrement"` option in **eas.json** to [manage versions for you](../build-reference/app-versions/#remote-version-source).
+
+</Collapsible>
+
+<Collapsible summary="nativeVersion">
+
+The `"nativeVersion"` policy is provided for projects with that wish to define their runtime compatibility based on the project's current `"version"` and `"buildNumber"` (iOS) or `"versionCode"` (Android) properties.
+
+For example, in a project that has the following in its app config:
+
+```json
+{
+  "expo": {
+    "runtimeVersion": {
+      "policy": "nativeVersion"
+    },
+    "version": "1.0.0",
+    "ios": {
+      "buildNumber": "1"
+    },
+    "android": {
+      "versionCode": 1
+    }
+  }
+}
+```
+
+The runtime version for the Android and iOS builds and any updates would be the combination of `"[version]([buildNumber|versionCode])"`, which in this case would be `"1.0.0(1)"`.
+
+This policy is great for projects that contain custom native code and that update the native version numbers (`"buildNumber"` for iOS and `"versionCode"` for Android) for each build. To submit an app, the app stores require an updated native version number for each submitted build, which makes this policy convenient if you want to be sure that every app uploaded to the Play Store's Internal Test Track and the App Store's TestFlight distribution tools has a different `runtimeVersion`.
+
+It's important to know that this policy does require management of the native version numbers manually between each build.
+
+Also, if you select a different native version between Android and iOS, you'll end up with builds and updates with separate runtime versions.
+
+</Collapsible>
+
+<Collapsible summary="fingerprint">
+
+> **warning** The `fingerprint` runtime policy is still in beta as of SDK 51 and not yet considered stable. Use may result in unexpected behavior.
```

Comment:
```suggestion
> **warning** The `fingerprint` runtime policy is still in beta since SDK 51 and not yet considered stable. Use may result in unexpected behavior.
```

Since this doc will be latest with SDK 52, let's update the callout above.